examples/misc/edit-sql-expressions.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Edit SQL & Expressions | CARTO</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="UTF-8">
+  <!-- Include CARTO GL JS -->
+  <script src="../../dist/carto-gl.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="../../vendor/mapbox-gl-dev.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="../../vendor/mapbox-gl-dev.css" rel="stylesheet" />
+  <style>
+    * {
+      margin:0;
+      padding:0;
+      box-sizing: border-box;
+    }
+    *:focus {
+      outline: none;
+    }
+    html {
+      height: 100%;
+    }
+    body {
+      background: #F2F6F9;
+      height: 100%;
+      font-family: 'Open sans'
+    }
+    #container {
+      display: flex;
+      height: 100%;
+    }
+    #map {
+      flex: 1;
+    }
+    #controls {
+      width: 308px;
+    }
+    .control-inner {
+      background: #fff;
+      padding: 20px 24px;
+      border-bottom: 2px solid #F2F6F9;
+    }
+    .control-inner {
+      font-size: 10px;
+      line-height: 14px;
+    }
+    .control-inner p {
+      margin-bottom: 4px;
+      color: #747D82;
+    }
+    .control-inner p:last-child {
+      margin-bottom: 0;
+    }
+    .control-inner h3 {
+      font-weight: 400;
+      font-size: 16px;
+      line-height: 22px;
+      margin-bottom: 8px;
+    }
+    .button-update {
+      border: 1px solid #1785FB;
+      color: #1785FB;
+      font-size: 12px;
+      line-height: 16px;
+      padding: 4px 12px;
+      border-radius: 4px;
+      font-family: 'Open sans';
+      cursor: pointer;
+      text-transform: uppercase;
+      font-weight: 600;
+    }
+    .button-update:hover {
+      background: rgba(23,133,251,.08);
+    }
+    textarea {
+      height: 100px; width: 100%; resize: none;
+      padding: 7px 8px 6px;
+      border: 1px solid #DDD;
+      border-radius: 4px;
+      font-size: 12px;
+      line-height: 16px;
+      font-family: 'Open Sans';
+      color: #6A6A6A;
+      margin-bottom: 8px;
+    }
+    .action-container {
+      display: flex;
+      justify-content: flex-end;
+    }
+  </style>
+</head>
+<body>
+  <div id="container">
+    <div id="map"></div>
+    <div id="controls">
+      <div class="control-inner">
+        <h3>SQL</h3>
+        <textarea id="sql" rows="3">
+SELECT * FROM ne_10m_populated_places_simple
+        </textarea>
+        <div class="action-container">
+          <button class="button-update" onclick="updateSource()">
+            <span>UPDATE SQL</span>
+          </button>
+        </div>
+      </div>
+      <div class="control-inner">
+        <h3>Expressions</h3>
+        <textarea id="expressions" rows="3">
+width: 6
+color: rgba(0.93,0.30,0.35,1.0)
+strokeWidth: 0.5
+strokeColor: rgba(1,1,1,1)
+          </textarea>
+          <div class="action-container">
+            <button class="button-update" onclick="updateStyle()">
```

Comment:
I think a `blend  style` button would be nice so the users can see the diference between a [blend](https://cartodb.github.io/renderer-prototype/docs/public/carto.Layer.html#blendToStyle) and a [update](https://cartodb.github.io/renderer-prototype/docs/public/carto.Layer.html#setStyle).